src/containers/month.js

```diff
@@ -0,0 +1,26 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import TaskGroup from './task-group';
+
+const Month = props => (
+  <li key={props.abbr} className='months-list__item'>
+    <div className='months-list-item__label-container'>
+      <span className='months-list-item__label' onClick={() => this.props.onMonthSelect}>
```

Comment:
Shouldn't it be `onClick={() => this.props.onMonthSelect()}` (invoking onMonthSelect)? Or just `onClick={this.props.onMonthSelect}`, if that works. Unless React is doing something odd, this looks like it wouldn't actually call the handler.